tests/db_engine_specs/presto_tests.py

```diff
@@ -490,3 +491,24 @@ def test_presto_expand_data_array(self):
         self.assertEqual(actual_cols, expected_cols)
         self.assertEqual(actual_data, expected_data)
         self.assertEqual(actual_expanded_cols, expected_expanded_cols)
+
+    @staticmethod
+    def test_get_sqla_column_type():
+        sqla_type = PrestoEngineSpec.get_sqla_column_type("varchar(255)")
```

Comment:
it would be nice to add some more test cases here especially the ones that are not supported by presto

Reply:
Since this is blocking another fairly critical feature I'll defer the more comprehensive tests to a forthcoming PR and just cover the discovered bug here. I'm hoping I can carve out some time to build a template for testing this kind of functionality, and don't mind doing the initial work on the Presto spec.